woocommerce/woocommerce
# Show APMs on the new Payments menu

### All Submissions:






### Changes proposed in this Pull Request:

The changes in this pull request add the additional payment methods, such as PayPal, AmazonPay, Klarna, and Affirm to the WC Payments welcome page (experiment). If toggled, it allows installation of those extensions alongside with WCPay.

Implemented the following items according to the design (7QOkvxzz1ylS3QBS4zqW8R-fi-10292%3A40323):
- Added a new Additional Payment Methods block to the WC Payments welcome page experiment.
- Removed the big FAQ block and left only a small row - Get in touch if the user has more questions.
- Added a `wcpay_extension_installed` event on `Install` button click.

Closes https://github.com/Automattic/woocommerce-payments/issues/4529.

**Important**: installation of the Affirm plugin is not possible yet, because it is absent in wp.org. There is an ongoing discussion here (pc2DNy-166-p2). For now, it's out of scope (paJDYF-4Ov-p2#comment-13959).

### How to test the changes in this Pull Request:
0. Comment out [the following lines](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Internal/Admin/WcPayWelcomePage.php#L31-L74) and build the zip from this branch.
1. Create a website using https://jurassic.ninja/specialops/ (include only WC Payments Dev Tools, remove Jetpack that is enabled by default)
2. Go to WC Payments Dev Tools and unclick the Proxy WPCOM requests checkbox and save (otherwise we'll have issues like `can't resolve docker.internal`)
3. Install the WooCommerce plugin from the built zip
4. Activate it and pass the WooCommerce onboarding (don't Install WCPay - untick Add recommended business features to my site on the step 4)
5. Choose Storefront theme, since it's the most popular one
6. Enable the tracking in settings (Woocommerce -> Settings -> Advanced -> [WooCommerce.com](http://woocommerce.com/) : Allow usage of WooCommerce to be tracked)
7. Then visit `/wp-admin/admin.php?page=wc-admin&path=%2Fwc-pay-welcome-page`
Make sure it looks like at the screenshot below and matches the design
![image](https://user-images.githubusercontent.com/79862886/189925477-3bfc8fc5-5a0e-4876-82f0-ff1290d3abdc.png)
8. Check the toggles: click on each toggle and observe as the notice above appears and the text changes depending on the enabled plugins. Notice shouldn't appear if all toggles are off.
![image](https://user-images.githubusercontent.com/79862886/189925703-9929ab8c-80cc-4857-8113-15f5839bbac3.png)
9. Tick paypal, amazonpay and klarna, click Install. The extensions should be installed alongside WC Pay.
10. Approve Jetpack prompt
11. Complete Stripe KYC, you'll be redirected to payments overview screen. Verify that promotion was applied.
12. Verify that all the selected plugins were installed
13. Check that `wcadmin_wcpay_extension_installed` event was recorded (after 5 mins when tracking event will show up)

### Other information:




### FOR PR REVIEWER ONLY:
